example/server.js

```diff
@@ -2,22 +2,51 @@
 
 var Hapi = require('hapi');
 var Hoek = require('hoek');
-var tags = require('../lib/index.js');
+var fs = require('fs');
+var pg = require('pg');
 
 var server = new Hapi.Server();
+var tagsData = JSON.parse(fs.readFileSync('./example/tags.json').toString());
+var tags = require('../lib/index.js');
+var tagsPool = new pg.Pool();
```

Comment:
If we pass the config in the init function we need to create the pool in the function also:
```
function init (port, config, callback) {
  var tagsPool = new pg.Pool();
...
```